src/ampJson.js

```diff
@@ -1,7 +1,14 @@
-export default function ampJson(data, component) {
+export default function ampJson(data, component, attributes={}) {
   if (!data || !component) return '';
+
+  const attributesString = attributes ?
+    Object.entries(attributes).map(([key, val]) => (
+      `${key}=${val}`
+    )).join(' ').trim() :
+    undefined;
+
   return Object.entries(data).map(([key, value]) => (
-    `<${component} id="${key}">
+    `<${component} id="${key}" ${attributesString}>
```

Comment:
@ljharb 

So I just need to set the type of the component, what do you think about this solution:

```
export default function ampJson(data, component, type) {
  if (!data || !component) return '';

  return Object.entries(data).map(([key, value]) => (
    `<${component} id="${key}" type=${type}>
      <script type="application/json">
      ${JSON.stringify(value)}
      </script>
    </${component}>`
  )).join('\n');
}

```